addons/website_profile/controllers/main.py

```diff
@@ -259,16 +257,38 @@ def view_all_users_page(self, page=1, **kwargs):
                 user['karma_gain'] = user_data.get('karma_gain_total', 0)
             user_values.sort(key=itemgetter('position'))
 
+            if (uid not in users.ids):
+                # Need to keep the dom to search only for users that appear in the ranking page
+                current_user = User.sudo().search(expression.AND([[('id', '=', uid)], dom]))
```

Comment:
I cannot see my own rank if i'm not published and with karma > 1?

Reply:
If I well understood the code, the rankings are calculated between users that have their profile published and have > 1 karma.
So, I guess, others are "unranked"  ? 
Because, if you have 1 karma or less, you're probably last in the ranking page with a lot of other users.
And even if you have more karma, but aren't published, because you're not in the actual ranking, putting you in will result to have 2 people at the same position.
I guess what I can do if you want, it's to add the current user to the all user list even if he's not published or its karma is <= 1 and then it could work.
You tell me ! Or @sbuhl tell me ? :D